snippets/language-d.cson

```diff
@@ -297,9 +288,6 @@ class ${1:MyException} : ${4:Exception} {
 	"with":
 		"prefix": "with"
 		"body": "with"
-	"scope":
-		"prefix": "scope"
-		"body": "scope"
 	"asm":
 		"prefix": "asm"
 		"body": "asm"
```

Comment:
the snippets can probably completely be removed, they are handled by workspace-d/serve-d now.

In code-d there are snippets for diet files which may be useful though.

Reply:
I need to test if the snippet provider works. For now, I think this does the work 🤔